src/matlab2tikz.m

```diff
@@ -2600,6 +2602,18 @@ function assertRegularAxes(handle)
             plotOptions = opts_add(plotOptions,'contour/labels','false');
         end
 
+        % Get line properties
+        [m2t, lineOptions] = getLineOptions(m2t, h);
```

Comment:
I cant follow you where is that code? In any case it should not make any difference.

Also i think you read the diff wrong. The "solid" options are removed from the code not inserted. Previously we would have splattered them everywhere.